WATOLINK/bci-gaming
# Implement Boss

## ClickUp ticket link

[Move boss to new scripts](https://app.clickup.com/t/8687zq5jq)



## Implementation description

Implements cactus boss scripts into `scenes/demos/Test New Boss`

Adds:
- Boss health bar
- Boss contact and ranged attacks
Fixes:
- Enemy health bar (also adds a outline if wanted)


## Steps to test
1. Open the Test New Boss scene and play around with boss
2. Does it feel like reasonable attacks? Anything else to add/change
3. Verify boss health bar looks good at top of screen, enemy health bar follows boss



## What should reviewers focus on?

What kind of health bar is better, outline or no outline?

Any other attacks to add/change/balance?

## Checklist
